### PR TITLE
INT-975 - fix pagantisIsEnabled warn

### PR DESCRIPTION
**Pull request by rmarinleal**
* fix pagantisIsEnabled warn

### CODE REVIEW

File: extension/app/code/community/Pagantis/Pagantis/Block/Product/Simulator.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ protected function _construct()
 
         if (in_array(strtolower($locale), $allowedCountries) &&
             $amount >= $pagantisMinAmount &&
-            ($amount <= $pagantisMaxAmount || $pagantisMaxAmount == '0')) {
+            ($amount <= $pagantisMaxAmount || $pagantisMaxAmount == '0') && $config['active']) {
```
Comment (pgarcess) [inline extension/app/code/community/Pagantis/Pagantis/Block/Product/Simulator.php]: && $config['active'] === '1'

File: extension/app/design/frontend/base/default/template/pagantis/product/simulator.phtml [hunk 2]
```diff
@@ -4,7 +4,9 @@
  * @var bool $enabled
  * @var string $publicKey
  */
-if ($pagantisIsEnabled && $pagantisSimulatorIsEnabled && $pagantisPublicKey != '') {
+if (isset($pagantisIsEnabled) && $pagantisIsEnabled &&
```
Comment (pgarcess) [inline extension/app/design/frontend/base/default/template/pagantis/product/simulator.phtml]: && $pagantisIsEnabled === '1'

File: extension/app/code/community/Pagantis/Pagantis/Block/Product/Simulator.php (hunk 1 above)
Comment (rmarinleal) [reply extension/app/code/community/Pagantis/Pagantis/Block/Product/Simulator.php]: changed

File: extension/app/design/frontend/base/default/template/pagantis/product/simulator.phtml (hunk 2 above)
Comment (rmarinleal) [reply extension/app/design/frontend/base/default/template/pagantis/product/simulator.phtml]: changed

**Review by pgarcess — APPROVED** [review]

**Pull request merged**